### DISCUSSION TITLE
TTGO LoRA 868/915 MHz pair setup

### DISCUSSION THREAD

**Discussion by arizonajon1** [richonguzman/LoRa_APRS_iGate#209] [General]
Feliz Navidad and happy holidays -

Thank you for such a nice installer for both APRS IGate and for APRS tracker. I have two Lilygo TTGO vintage 20210104 T3_V1.6.1 868/915 MHz devices, OLED, no GPS.

I set up one unit as an iGate, it is now a standalone device connected to my local Wi-Fi network. On the OLED screen, I can see 

N7UV-12
WiFi: OK   APRS-IS: OK 
IP: 192.168.1.218
Stations (30min) = 0
APRS-IS -----> LoRA TX
W7ARA-4 > OBJECT 
listening...

And occasionally the object retransmitted from APRS-IS changes, so that seems to be working properly for the iGate. 

I have not touched the radio frequency used, so right now it is still configured for 437 MHz even those this is a 915 MHz device. If I try to configure radio frequency to a 902-928 MHz frequency, the device locks up and i have to go through the entire flashing process again.

Any idea on what I'm doing wrong?

Cheers and 73 - Jon N7UV
Phoenix Arizona USA

**Comment by richonguzman** [comment]
Hi

Share please the firmware board you selected first. Maybe you did not select the 915mhz version of the board you need to avoid the locking up

**Reply by arizonajon1** [reply]
Good day!
If you mean the items selected on the CA2RXU "LORA APRS IGATE - WEB FLASHER" installer page, I selected 
LoRa32 V2.1/1.6.1 (915 MHz)
V2.1.3 (06 December 2024)
First flash or Factory reset (web_factory.bin)

Then, I select "FLASH  FIRMWARE"

and am presented with the following:
Install ttgo-lora32-v21_915 factory.json
Logs & Console

I select Install ttgo-lora32-v21_915 factory.json
I select "Erase Device"

and confirm that I want to install 
ttgo-lora32-v21_915 factory.json V2.1.3

and it begins installation.

Is that instructive?

Cheers and 73 - Jon N7UV
Phoenix, Arizona, USA

**Reply by richonguzman** [reply]
So you select the right lora32 ttgo board + 915MHz

Then flash then configure for 915mhz freq in webconfig page

And it works or not?

**Comment by arizonajon1** [comment]
This is what is presented to me. Only 433775000 Hz. If I modify it to 915000000 Hz, for instance, it will fail the reboot cycle.

![image](https://github.com/user-attachments/assets/aaf1d89c-15d1-46bd-bac5-862b060c124f)

**Reply by arizonajon1** [reply]
Thank you for your support!

Here are two pictures of the boards that I have

![IMG_5854](https://github.com/user-attachments/assets/3f5de528-d97c-4973-a2dc-4eac999272fa)

![IMG_5855](https://github.com/user-attachments/assets/92c3da45-3b74-432a-9cd0-e735e9d90b28)

**Comment by richonguzman** [comment]
If you flash 915 and change freq to 915000000 it should work right away

So I suspect your board is not 915. Can you share a picture of it?

**Reply by arizonajon1** [reply]
Thanks for your patience - pictures of the boards are included inline above

**Comment by richonguzman** [comment]
i see both boards as 915, and you flashed 915 versions of Lora32 board. So it should work with 915 MHz freq

have you tested further?

**Reply by arizonajon1** [reply]
Not much. Each time I attempt to change the frequency from the 433 MHz  default to any other frequency (even 434 MHz) it crashes the firmware reboot. Am I correct to assume that the 915 firmware image would populate the frequency text boxes with 915 MHz frequencies?

**Reply by richonguzman** [reply]
You would have to change the freq as it's not changed automatically

**Reply by arizonajon1** [reply]
Wow, I made a lot of progress and still don't know what it was that was preventing me from changing frequencies. Sorry for all the issues!
I am now to the point that I have a unit running 927500000 Hz as a digi/igate on a 90 deg sector antenna taped into the house window, it has a clear view to the sw thru nw. It "seems" to be passing object messages from APRS-IS to RF, at least according to the OLED display. Not very often, but objects like W7ARA-4, NK3H-S, etc., are showing up on the OLED. The NK3H-S also seems to show -103 dBm and 7.5 dB SNR.
The other unit is in the truck on a roof-mounted 915 antenna and connected via USB cable to a truck-installed W11 PC running YAAC. The YAAC PC recognizes the TTGO board on COM 7. I've set up YAAC to look at COM7 as running as a serial KISS connection. What is the serial baud rate supposed to be? 9600? I've tried cycling through baud rates on YAAC COM7 port test, but haven't seen anything that is more than typical baud rate mismatch gibberish.

**Reply by richonguzman** [reply]
lora serial speed is 115200, maybe test this for YAAC

**Reply by arizonajon1** [reply]
ok, just set serial speed to 115200. I shut down the 144 MHz APRS TNC connected to the truck YAAC, so for the YAAC mobile there's only serial GPS and serial KISS to the TTGO. 

Truck is still parked in driveway and so "mobile" TTGO (N7UV-13) is connected via Wi-Fi to the house network. I forced a beacon transmit from the TTGO digi/igate and the screen shot is what occurred. There does seem to be at least APRS-IS  > N7UV-12 TTGO digi/igate > RF happening now.

![image](https://github.com/user-attachments/assets/fdb57ee4-af1f-4cd8-be8a-5e71efc959d9)

and 

![image](https://github.com/user-attachments/assets/6a1f4383-f698-4de0-96fa-916739672dd7)

This seems to be a one-way success. It doesn't look like the YAAC > N7UV-13 TTGO mobile > RF > N7UV-12 TTGO digi/igate is working, as the YAAC raw packet sniffer isn't indicating any raw messages seen from N7UV-13 (the mobile). And aprs.fi isn't reporting hearing N7UV-13.

**Reply by arizonajon1** [reply]
My truck YAAC N7UV-13 has serial_TNC enabled as follows:

![image](https://github.com/user-attachments/assets/eb08ee67-9478-4291-885b-7f3e250f4cc7)

**Reply by richonguzman** [reply]
so at 115200 does it work for you?

**Reply by richonguzman** [reply]
so 13 is sending from YAAC? I got lost with this

**Reply by arizonajon1** [reply]
115200 appears to work.

Sorry for the confusion. -13 is the mobile, -12 is the base. 
YAAC in the truck has TTGO 915 connected via USB COM7.

**Comment by arizonajon1** [comment]
How do I interpret this?
![image](https://github.com/user-attachments/assets/3ea64cd4-eca5-4dda-9de2-62947b3b8ac2)
the -13 in the truck should not be connecting to APRS-IS, it should be connected only via RF as shown here:
![image](https://github.com/user-attachments/assets/2a5f615d-60fe-4ee2-a6bb-50613d5da78c)

![image](https://github.com/user-attachments/assets/b5f36295-dc26-408b-8e12-93c96c20e622)
This is the way I have -13 configured.

**Reply by richonguzman** [reply]
-13 is mobile, but....  you have it as Igate...  is this what you want?

I mean, tracker firmware works better for moving stations, but if you have YAAC on mobile its ok , I guess

**Reply by arizonajon1** [reply]
happy new year!
That's where more of my confusion is. I had originally built it as a tracker, but it didn't have a built-in gps and would need to act as a YAAC client. Should i go back to the tracker build?

**Reply by richonguzman** [reply]
if the tracker is used as TNC over Bluetooth it could receive from phone (aprsdroid or aprs.fi app) or a PC app as well

**Reply by arizonajon1** [reply]
Adding bluetooth connections to other devices seems to be adding complexity where it is not required. 

My desire is to have a peer-peer connection like normal ax.25 and APRS so that the mobile unit can act as a virtual TNC for apps like YAAC.
Also, there are situations where I go hiking into areas that have no coverage, but with the truck parked there and it acting as a relay/digi I can carry something like a "tracker" which would communicate to the node in the truck and that node then digipeat the message to the network. With APRS, I have a truck-mounted Microsat WX3in1 that does exactly that.
Is there a way to mimic this using your toolset?

**Reply by arizonajon1** [reply]
> -13 is mobile, but.... you have it as Igate... is this what you want?
> 
> I mean, tracker firmware works better for moving stations, but if you have YAAC on mobile its ok , I guess

I just reviewed the configuration for -13 (truck) and don't see what you are saying about configuration as an igate.
![image](https://github.com/user-attachments/assets/37099534-3af6-44d0-9791-0a7383e16152)

Perhaps I'm just misinterpreting the setup parameters, but it appears to me that I've explicitly told the firmware to send beacons only by RF.

Also, in the section for APRS-IS configuration, I have the following for -13:
![image](https://github.com/user-attachments/assets/e03653a4-3609-4fcc-be12-0820e5c4471a)
so I don't know how i have it configurated as igate. Can you help me to understand?

**Comment by arizonajon1** [comment]
The issues I raised have been resolved. Thanks Ricardo!
